### ISSUE TITLE
Cannot login to Nextcloud instance on Android

### ISSUE THREAD

**Issue by sycam0r-e** [floccusaddon/floccus#1420]
### Which version of floccus are you using?

4.19.0

### Sync method

Nextcloud Bookmarks

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

Andoid 14 - GrapheneOS

### Which version of Nextcloud Bookmarks are you using? (if relevant)

13.1.1

### Which version of Nextcloud? (if relevant)

26.0.7

### Describe the Bug

When trying to set up Floccus on my Android phone, after selecting "Nextcloud Bookmarks" in the first step, I cannot click "Log in" or "Alternative log in using app token" in the next step.

![bug](https://github.com/floccusaddon/floccus/assets/98783999/ab3df1d7-9d81-4bc4-8a7a-8f40e1010852)

### Expected Behavior

Be able to select either select "Log in" or "Alternative log in using app token" in step two.

### To Reproduce

1. Open Floccus.
2. Specify my up and running Nextcloud URL.
3. On the next step, try to click "Log in" or "Alternative log in using app token".

### Debug log provided

- [ ] I have provided a debug log file

**Comment by marcelklehr** [comment]
Mmh, can you provide a screenshot?

**Comment by joshtrichards** [comment]
Might be related: nextcloud/server#41098

**Comment by sycam0r-e** [comment]
On this screen, I cannot click "Log in" nor "Alternative log in using app token".

<img src='https://github.com/floccusaddon/floccus/assets/98783999/c30e516e-848e-4faa-9967-dc1e6f88e606'  width=40% height=40%>

**Comment by marcelklehr** [comment]
Are you able to setup the nextcloud desktop client?

**Comment by sycam0r-e** [comment]
Yes, the nextcloud client apps (both on Mac and Android) are running fine.  

Thanks for the link!
